laterpay/asset_sources/scss/components/_purchase_button.scss

```diff
@@ -90,6 +90,13 @@
     }
 }
 
+// wrapper used to set the purchase button apart from the post content when positioned automatically
+.lp_purchase-button-wrapper {
+    @extend %clearfix;
+
+    margin: $fs 0 $fs--2 !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces